### PR TITLE
Gotta go fast

### PR DESCRIPTION
**Pull request by DamianX**
- json_encode was 40% faster at parsing JSON than json_reader
- removed minimap hash generation
- minimaps are only generated if the current git revision is different from last round's (or if there's no cached file, duh)

Prepare for unforeseen consequences.

:cl:
 * tweak: Made the server start a bit faster.

### CODE REVIEW

**Comment by Kharkov72** [comment]
inb4 this completely breaks the juke

**Comment by DamianX** [comment]
It works on MY machine.

File: code/modules/html_interface/map/interactive_map.dm [hunk 1]
```diff
@@ -81,12 +81,26 @@ var/const/ALLOW_CENTCOMM = FALSE
 /proc/generateMiniMaps()
 	set category = "Debug"
 	set name = "Generate minimaps"
-	//spawn // NO
+
+	var/last_git_hash_path = "data/last_git_commit_hash.txt"
```
Comment (ShiftyRail) [inline code/modules/html_interface/map/interactive_map.dm:85]: This is a new file being generated, right ? I think "data" is added to the gitignore, but can you confirm this for me, please ?

Comment (DamianX) [reply code/modules/html_interface/map/interactive_map.dm:85]: see latest commit

**Review by ShiftyRail — COMMENTED** [review]
Looks good for me

**Comment by SonixApache** [comment]
when

**Comment by DamianX** [comment]
bump

**Comment by DamianX** [comment]
Feel free to take the code from this branch and open a new PR. It's better if someone else does it anyway since I have no intention of dealing with eventual issues this might cause at the moment.

**Comment by SonixApache** [comment]
😢

**Comment by ShiftyRail** [comment]
👀

**Comment by DeityLink** [comment]
👀

**Comment by Eneocho** [comment]
something something consequences.

**Comment by SonixApache** [comment]
WHEN

**Pull request merged**